src/components/Contentful/_contentful_highlight.scss

```diff
@@ -72,4 +72,17 @@
       }
     }
   }
+
+  .contentful-highlight__author {
+    span {
+        height: 45px;
+        font-size: 14px;
```

Comment:
font-size and line-height need to be in rems and these should already exist in the contentful_settings scss. also, the properties with value "normal" don't need to be defined.